bundles/binding/org.openhab.binding.ecotouch/src/main/java/org/openhab/binding/ecotouch/internal/EcoTouchBinding.java

```diff
@@ -163,10 +192,8 @@ private void handleEventType(org.openhab.core.types.State state,
 	/**
 	 * @{inheritDoc
 	 */
-	@Override
```

Comment:
my java compiler (not sure which version... Java8?) failed compiling this. There is no void updated(...) in the base classes to @Override